webapp/src/onboarding.ts

```diff
@@ -0,0 +1,76 @@
+export interface querySelector {
+    targetQuery: string;
+    sansQuery?: string;
+    sansLocation?: pxt.tour.BubbleLocation;
+}
+
+function getTargetMap(target: string): querySelector {
+    const targetMap: pxt.Map<querySelector> = {
+        "simulator": {
```

Comment:
Since it might not be obvious, the reason I suggested doing it this way instead of directly putting the query / etc in the markdown was to avoid the content authors getting too deep into the weeds / boxing us in -- query structures rely on the structure of the dom which is basically an implementation detail, so this way we have a fixed set we can maintain compatibility with (so we don't e.g. get prevented from refactoring because we don't want to check every existing md file)

Reply:
Thanks for adding this info, Joey!